### PR TITLE
Updates chrome to version 90

### PR DESCRIPTION
**Pull request by lucalves**
Hello everyone, everything good?

In that PR, I updated the docs to version 90 of Chrome.

According to [Chrome Status](https://www.chromestatus.com/features/schedule), 92 is the nightly version.

😉

### CODE REVIEW

**Comment by vinyldarkscratch** [comment]
Looks like you've got some syntax errors in your PR.  Can you resolve them?

**Comment by lucalves** [comment]
I was unable to identify which line the syntax error, @vinyldarkscratch. Would you help me?

File: browsers/chrome.json [hunk 1]
```diff
@@ -616,20 +616,26 @@
...
-          "status": "nightly",
+          "status": "beta",
           "engine": "Blink",
           "engine_version": "91"
-        }
+        },
+        "92": {
+          "status": "nightly",
+          "engine": "Blink",
+          "engine_version": "92"
```
Comment (vinyldarkscratch) [inline browsers/chrome.json:638]:
```suggestion
          "engine_version": "92"
        }
```

File: browsers/chrome_android.json [hunk 2]
```diff
@@ -453,20 +453,26 @@
...
-          "status": "nightly",
+          "status": "beta",
           "engine": "Blink",
           "engine_version": "91"
-        }
+        },
+        "92": {
+          "status": "nightly",
+          "engine": "Blink",
+          "engine_version": "92"
```
Comment (vinyldarkscratch) [inline browsers/chrome_android.json:475]:
```suggestion
          "engine_version": "92"
        }
```

File: browsers/webview_android.json [hunk 3]
```diff
@@ -444,20 +444,27 @@
...
+          "status": "beta",
           "engine": "Blink",
           "engine_version": "91"
-        }
+        },
+        "92": {
+          "status": "nightly",
+          "engine": "Blink",
+          "engine_version": "92"
+      }
```
Comment (vinyldarkscratch) [inline browsers/webview_android.json:467]:
```suggestion
        }
```

**Review by vinyldarkscratch — CHANGES_REQUESTED** [review]

**Comment by lucalves** [comment]
I solved your suggestions, @vinyldarkscratch! Thanks.

File: browsers/chrome.json [hunk 4]
```diff
@@ -616,19 +616,26 @@
...
           "release_date": "2021-03-02",
           "release_notes": "https://chromereleases.googleblog.com/2021/03/stable-channel-update-for-desktop.html",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "89"
         },
         "90": {
-          "status": "beta",
+          "release_date": "2021-04-14",
```
Comment (vinyldarkscratch) [inline browsers/chrome.json:624]:
```suggestion
          "release_date": "2021-04-13",
```

**Review by vinyldarkscratch — APPROVED** [review]
Alrighty, everything's looking good to me, thank you for the PR!  I'll merge once the checks have passed!

**Pull request merged**